### PR TITLE
MAINT: Updated muFFT dependency to muFFT-0.9.1

### PR DESCRIPTION
**Pull request by pastewka**
- muFFT has now been release on PyPI
- Moved documentation from README.md to the docs folder

### CODE REVIEW

**Comment by pastewka** [comment]
@AntoineSIMTEK Can you review this? It will hopefully pass.

File: SurfaceTopography/ChangeLog.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
 =============================
 
-v0.90.0 (not yet released)
---------------------------
+v0.90.0 (17Jun20)
+-----------------
 
 - Refactored PyCo code into three separate Python modules:
   SurfaceTopography, ContactMechanics and Adhesion
+- muFFT dependency updated to muFFT-0.9.0
```
Comment (AntoineSIMTEK) [inline SurfaceTopography/ChangeLog.md:10]: 0.9.1

File: docs/contributing.rst [hunk 2]
```diff
@@ -104,4 +120,13 @@ happens. It is a good idea to let the plots ready for use:
 
 .. _PEP-8: https://www.python.org/dev/peps/pep-0008/
 .. _numpydoc: https://numpydoc.readthedocs.io/
-.. _runtests: https://github.com/AntoineSIMTEK/runtests
\ No newline at end of file
+.. _runtests: https://github.com/AntoineSIMTEK/runtests
```
Comment (AntoineSIMTEK) [inline docs/contributing.rst:123]: I guess we can change this to the main directory now

File: docs/installation.rst [hunk 3]
```diff
@@ -1,18 +1,76 @@
...
+    export LDFLAGS="-L/usr/local/opt/openblas/lib $LDFLAGS"
+    export CPPFLAGS="-I/usr/local/opt/openblas/include $CPPFLAGS"
+    export PKG_CONFIG_PATH="/usr/local/opt/openblas/lib/pkgconfig:$PKG_CONFIG_PATH"
+
+    export LDFLAGS="-L/usr/local/opt/lapack/lib $LDFLAGS"
+    export CPPFLAGS="-I/usr/local/opt/lapack/include $CPPFLAGS"
+    export PKG_CONFIG_PATH="/usr/local/opt/lapack/lib/pkgconfig:$PKG_CONFIG_PATH"
+
+where the paths have probably to be adapted to your particular installation method.
+Here FFTW3_ was installed via Homebrew_.
```
Comment (AntoineSIMTEK) [inline docs/installation.rst:64]: I think I meant openblas and lapack

File: docs/testing.rst [hunk 4]
```diff
@@ -1,10 +1,23 @@
...
 .. code-block:: bash
 
     pytest
 
-in the main source directory to run the automated tests.
+Tests that are parallelizable have to run with runtests_.
+
+.. code-block:: bash
+
+    python run-tests.py
```
Comment (AntoineSIMTEK) [inline docs/testing.rst:14]: python run-tests.py --no-build

File: .install_parallel_netcdf.sh [hunk 5]
```diff
@@ -4,19 +4,21 @@
 # broken on most distributions.
 #
 
-curl -k https://parallel-netcdf.github.io/Release/pnetcdf-${PNETCDF_VERSION}.tar.gz | tar -xzC ${BUILDDIR} &&
-  cd ${BUILDDIR}/pnetcdf-${PNETCDF_VERSION} &&
-  CC=mpicc CXX=mpicxx ./configure --disable-fortran --disable-cxx --enable-shared --prefix=${PREFIX} &&
-  make &&
-  make install &&
-  cd -
+# Installed via apt in .travis.yml
```
Comment (AntoineSIMTEK) [inline .install_parallel_netcdf.sh]: so the  apt-packages libpnetcdf-dev libhdf5-openmpi-dev are working properly now, only NetCDF needs to be compiled ourselves ? Is that generally true ?

**Comment by pastewka** [comment]
@AntoineSIMTEK I made those suggested changes.

File: .install_parallel_netcdf.sh (hunk 5 above)
Comment (pastewka) [reply .install_parallel_netcdf.sh]: No... it does not work, I reverted that change

**Pull request merged**